### PR TITLE
Dao 9 new

### PR DESCRIPTION
**Pull request by entertainment-veks**

### CODE REVIEW

File: src/main/java/taxiservice/controller/driver/LoginDriverController.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String login = req.getParameter("login");
+        String password = req.getParameter("password");
+
+        try {
+            Driver input = authService.login(login, password);
+
+            HttpSession session = req.getSession();
```
Comment (RomanDubovskyi) [inline src/main/java/taxiservice/controller/driver/LoginDriverController.java:34]:
```suggestion
        String password = req.getParameter("password");
        try {
            Driver input = authService.login(login, password);
            HttpSession session = req.getSession();
```

**Review by RomanDubovskyi — CHANGES_REQUESTED** [review]
fix comments from the chat

**Comment by RomanDubovskyi** [comment]
fix comments that i left in channel jv_nov20

File: src/main/java/taxiservice/web/filter/AuthentificationFilter.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        HttpServletResponse resp = (HttpServletResponse) response;
+
+        String url = req.getServletPath();
+        if (links.contains(url)) {
+            chain.doFilter(req, resp);
+            return;
+        }
+
+        Long driverId = (Long) req.getSession().getAttribute(DRIVER_ID);
+        if (driverId == null || driverService.get(driverId) == null) {
```
Comment (RomanDubovskyi) [inline src/main/java/taxiservice/web/filter/AuthentificationFilter.java:44]: can get() from driverService return null?

File: src/main/java/taxiservice/web/filter/AuthentificationFilter.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+import javax.servlet.http.HttpServletResponse;
+import taxiservice.lib.Injector;
+import taxiservice.service.DriverService;
+
+public class AuthentificationFilter implements Filter {
+    private static final String DRIVER_ID = "driver_id";
+    private static final Injector injector = Injector.getInstance("taxiservice");
+    private static DriverService driverService = (DriverService)
+            injector.getInstance(DriverService.class);
+    private Set<String> links = new HashSet<>();
```
Comment (RomanDubovskyi) [inline src/main/java/taxiservice/web/filter/AuthentificationFilter.java:22]:
```suggestion
    private Set<String> allowedLinks = new HashSet<>();
```

**Review by RomanDubovskyi — CHANGES_REQUESTED** [review]

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**